extensions/EXT/EGL_EXT_platform_xcb.txt

```diff
@@ -0,0 +1,357 @@
+Name
+
+    EXT_platform_xcb
+
+Name Strings
+
+    EGL_EXT_platform_xcb
+
+Contributors
+
+    Yuxuan Shui <yshuiv7@gmail.com>
+
+Contacts
+
+    Yuxuan Shui <yshuiv7@gmail.com>
+
+Status
+
+    Complete
+
+Version
+
+    Version 1, 2020-08-28
+
+Number
+
+    EGL Extension #141
+
+Extension Type
+
+    EGL client extension
+
+Dependencies
+
+    Requires EGL_EXT_client_extensions to query its existence without
+    a display.
+
+    Requires EGL_EXT_platform_base.
+
+    This extension is written against the wording of version 9 of the
+    EGL_EXT_platform_base specification.
+
+Overview
+
+    This extension defines how to create EGL resources from native X11
+    resources using the functions defined by EGL_EXT_platform_base.
+
+    The native X11 resources required by this extension are xcb resources.
+    All X11 types discussed here are defined by the header `xcb.h`.
+
+New Types
+
+    None
+
+New Procedures and Functions
+
+    None
+
+New Tokens
+
+    Accepted as the <platform> argument of eglGetPlatformDisplayEXT:
+
+        EGL_PLATFORM_XCB_EXT                    0x31DC
+
+    Accepted as an attribute name in the <attrib_list> argument of
+    eglGetPlatformDisplayEXT:
+
+        EGL_PLATFORM_XCB_SCREEN_EXT             0x31DE
+
+Additions to the EGL Specification
+
+    None.
+
+New Behavior
+
+    To determine if the EGL implementation supports this extension, clients
+    should query the EGL_EXTENSIONS string of EGL_NO_DISPLAY.
+
+    This extension defines the same set of behaviors as the X11 platform
+    extension, except Xlib types are replaced with xcb types.
+
+    To obtain an EGLDisplay backed by an X11 screen, call
+    eglGetPlatformDisplayEXT with <platform> set to EGL_PLATFORM_XCB_EXT. The
+    <native_display> parameter specifies the X11 display connection to use, and
+    must point to a valid xcb `xcb_connection_t` or be NULL.  If <native_display>
```

Comment:
NULL, or EGL_DEFAULT_DISPLAY?

Reply:
This is the same language used in `EXT_platform_x11`.
Though I think `EGL_DEFAULT_DISPLAY` is the better one here.

Reply:
Ah, yes, it would probably be best to update both, but at least we can get it right here to start.